middlewares/validators/user.js

```diff
@@ -158,6 +159,9 @@ async function getUsers(req, res, next) {
       id: joi.string().optional().messages({
         "string.empty": "id value must not be empty",
       }),
+      discordId: joi.string().optional().messages({
```

Comment:
messages object. The string.empty message is only triggered if the discordId value is empty. However, the optional() modifier means that the discordId value is allowed to be empty. This means that the string.empty message will never be triggered.